src/components/WebexActivityStream/__snapshots__/WebexActivityStream.test.js.snap

```diff
@@ -1108,3 +1108,13 @@ exports[`Webex Activity Stream component Webex Activity Stream snapshots matches
   />
 </div>
 `;
+
+exports[`Webex Activity Stream component Webex Activity Stream snapshots matches with time rulers stream 1`] = `
```

Comment:
snapshot is not correct.

Reply:
![img](https://media0.giphy.com/media/eP1fobjusSbu/giphy.gif)